imageops.lisp

```diff
@@ -191,7 +191,8 @@
   (let ((len (reduce #'* (array-dimensions array))))
     (if force-simple
         (let* ((disp (make-array len
-                                 :displaced-to array))
+                                 :displaced-to array
+				 :element-type (array-element-type array)))
```

Comment:
@slyrus This looks like an optimization - the created array will not be a simple-array but will be specialized to the AET of the original array.